T2T Genomes Unveil Centromere Architecture and Adaptive Divergence in Large Yellow Croaker (Larimichthys crocea)


The large yellow croaker (Larimichthys crocea) is a key aquaculture species, yet gaps in high-quality genomes hinder studies of centromeric and adaptive evolution. This study presents two telomere-to-telomere (T2T), gapless genomes from the Min-Yuedong (MYD) and Daiqu (DQ) populations, which diverged 4.18 million years ago. The centromeres are characterized by a 42 bp tandem repeat (Cen-42) and invasions by LTR/ERV1 elements. Both populations exhibit transcriptional activity in centromeric regions, but display significant divergence in gene number and composition, likely driven by rapid population expansion and selective pressure. In addition, 5S rRNA genes form ultra-large tandem repeat clusters in the short-arm regions of more than 10 chromosomes. The T2T genome assemblies resolve previously unassembled regions, identifying 533 and 351 new genes in T2T-MYD and T2T-DQ, respectively. We identified 4 population-specific structural variations, located in pla2g4a, eno2, ptprb, and itrp3. Comparative genomic analyses highlight distinct adaptive features between the two populations, including differences in metabolic efficiency, arachidonic acid metabolism, chemosensasory function, and circadian rhythm. Collectively, these findings deepen our understanding of L. crocea evolution and provide valuable genomic resources for its conservation and breeding.



<img width="744" height="647" alt="image" src="https://github.com/user-attachments/assets/73b16a09-38cd-40c1-98e6-d17331eb446b" />
